storage/client_test.go

```diff
@@ -498,7 +498,6 @@ func (t *multiTestContextKVTransport) SendNext(done chan<- kv.BatchCall) {
 		done <- kv.BatchCall{Reply: br, Err: nil}
 	}) != nil {
 		done <- kv.BatchCall{Err: roachpb.NewSendError("store is stopped")}
-		t.mtc.expireLeases()
```

Comment:
> If a store is stopped, DistSender will send a request to another store (if one is available) and we'll get a NotLeaseHolderError which will expire the lease.

 OK, I think in that case this is safe. We've gotten a lot better about this kind of thing since this code was written. LGTM.